app/models/user.rb

```diff
@@ -0,0 +1,13 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
- [ ] [OPTIONAL] I would like to recommend that you enable `:confirmable` because it adds an additional layer of security to your application by requiring users to verify their email addresses. After users register, they receive a confirmation email with a link. They must click the link to confirm their email address and activate their accounts.